integration/it-integration/src/main/java/se/inera/intyg/rehabstod/integration/it/stub/RSTestIntygStub.java

```diff
@@ -51,10 +57,21 @@
 
     @PostConstruct
     public void init() {
-        intygsData = sjukfallIntygDataGenerator.generateIntygsData(numberOfPatients, intygPerPatient);
+        var stubData = rsTestDataGenerator.generateIntygsData(numberOfPatients, intygPerPatient);
+        intygsData = stubData.getIntygsData();
+        luCertificateData = stubData.getDiagnosedCertificates();
```

Comment:
What does LU stand for?

Reply:
Läkarutlåtande, i.e. luse, luae_fs and luae_na.

Reply:
Ny sektion för vanliga förkortningar tillagd i https://inera.atlassian.net/wiki/spaces/IT/pages/408257507/Riktlinjer+f+r+utveckling+i+Intygstj+nster